ontology/ontoavida.md

```diff
@@ -30,3 +30,12 @@ tags:
 tracker: https://gitlab.com/fortunalab/ontoavida/-/issues
 activity_status: active
 ---
+
+# OntoAvida
```

Comment:
this is redundant since the ontology page shows this based on the structured metadata

Reply:
Removing ...